### PR TITLE
parmed - Parameter/topology editor

### PR DESCRIPTION
**Pull request by chrisbarnettster**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by simonbray** [comment]
Confirming I'm happy to be listed as a maintainer.

**Comment by chrisbarnettster** [comment]
@tsenapathi

**Comment by tsenapathi** [comment]
Confirming I'm happy to be a maintainer.

File: recipes/parmed/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+build:
+  number: 0
+  entry_points:
+    - parmed = parmed.scripts:clapp
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: true  # [win or osx]
+
+requirements:
+  host:
+    - gcc_linux-64
```
Comment (mariusvniekerk) [inline recipes/parmed/meta.yaml]:
```suggestion
    - {{ compiler('c') }}
```

File: recipes/parmed/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    - gxx_linux-64
```
Comment (mariusvniekerk) [inline recipes/parmed/meta.yaml]:
```suggestion
    - {{ compiler('cxx') }}
```

File: recipes/parmed/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    - gfortran_linux-64
```
Comment (mariusvniekerk) [inline recipes/parmed/meta.yaml]:
```suggestion
    - compiler('fortran')
```

File: recipes/parmed/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    - libgcc-ng
```
Comment (mariusvniekerk) [inline recipes/parmed/meta.yaml]: Remove this
```suggestion
```

File: recipes/parmed/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    - pip
+    - python>=3.5
```
Comment (mariusvniekerk) [inline recipes/parmed/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/parmed/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  run:
+    - numpy >=1.14
+    - pandas
+    - python>=3.5
```
Comment (mariusvniekerk) [inline recipes/parmed/meta.yaml]:
```suggestion
    - python >=3.5
```

File: recipes/parmed/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,70 @@
+{% set name = "parmed" %}
+{% set version = "3.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/ba/a2/d6a9135efd6b1f7d70b79019a2fb2cd1472fceed979e66deaba64e2b641c/ParmEd-3.2.0.tar.gz"
```
Comment (mariusvniekerk) [inline recipes/parmed/meta.yaml]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

**Pull request merged**